pyproject.toml

```diff
@@ -49,35 +50,38 @@ chardet = "^5.2.0"
 docker = "^4.4.4"
 gitpython = "^3.1.2"
 jsonc-parser = "^1.1.5"
-jsonschema = "^4.19.0"
+jsonschema = "^4.22.0"
 openpyxl = "^3.1.2"
 opensearch-py = "^2.0.1"
 pyOpenSSL = "^23.1.1"
 PyJWT = "^2.6.0"
-pyramid = "1.10.4"
+#pyramid = "^2.0.2"
+pyramid = "1.10.8"
```

Comment:
Please use version >2 if at all possible, it has features in it we need upstream